torch_geometric/nn/to_hetero_module.py

```diff
@@ -9,42 +9,96 @@
 from torch_geometric.typing import EdgeType, NodeType, OptTensor
 from torch_geometric.utils import scatter
 
+from .dense import HeteroLinear, Linear
+
+
+class SubParam():
+    def __init__(self, param: torch.nn.Parameter, idx: int):
+        self.idx = idx
+        self.param = param
+
+    def get_data(self):
+        return self.param[self.idx].data
+
+    def set_data(self, data):
+        if self.param.data[self.idx].shape == data.t().shape:
+            self.param.data[self.idx] = data.t()
+        else:
+            self.param.data[self.idx] = data
+
+    data = property(get_data, set_data)
+
+
+class DummyLinear():
```

Comment:
Can you clarify `DummyLinear` for me?

Reply:
it exists as a  pythonic hack to support this usage from `test_to_hetero_and_rgcn_equal_output`:
```
        model.lin[node_type].weight.data = conv.root.data.t()
        model.lin[node_type].bias.data = conv.bias.data
```
Since we have a 3D parameter in the pyg-lib heterolinear instead of a list of Linear modules, its very hard to access the weight and bias of each node type in the way done in the test but I believe a user should still be able to do this.